compiler/src/dotty/tools/dotc/transform/PCPCheckAndHeal.scala

```diff
@@ -176,8 +176,10 @@ class PCPCheckAndHeal(@constructorOnly ictx: Context) extends TreeMapWithStages(
   /** If the type refers to a locally defined symbol (either directly, or in a pickled type),
    *  check that its staging level matches the current level.
    *  - Static types and term are allowed at any level.
-   *  - If a type reference is used a higher level, then it is inconsistent. Will attempt to heal before failing.
-   *  - If a term reference is used a different level, then it is inconsistent.
+   *  - If a type reference is used a higher level, then it is inconsistent.
+   *    Will attempt to heal before failing.
+   *  - If a term reference is used a higher level, then it is inconsistent.
+   *    It can not be healed because the term will not exists in the any future stage.
```

Comment:
"can not" is usually spelled "cannot", and should "the any" be "any"?
```suggestion
   *    It cannot be healed because the term will not exists in any future stage.
```